src/PKSim.Core/Batch/PopulationSettings.cs

```diff
@@ -113,8 +113,8 @@ public PopulationSettings()
          MaxHeight = double.NaN;
          MinBMI = double.NaN;
          MaxBMI = double.NaN;
-         MinGestationalAge = CoreConstants.PRETERM_RANGE.Min();
-         MaxGestationalAge = CoreConstants.PRETERM_RANGE.Max();
+         MinGestationalAge = CoreConstants.PretermRange.Min();
+         MaxGestationalAge = CoreConstants.PretermRange.Max();
          NumberOfIndividuals = CoreConstants.DEFAULT_NUMBER_OF_INDIVIDUALS_IN_POPULATION;
```

Comment:
Had to change this because `PKSimCreatePopulation `in the Matlab-Toolbox crashes otherwise
(and I didn't want crate new Matlab-Toolbox-Core and Matlab-Toolbox version for this hotfix release)